services/web/server/src/simcore_service_webserver/projects/_nodes_handlers.py

```diff
@@ -105,6 +106,12 @@ async def wrapper(request: web.Request) -> web.StreamResponse:
             raise web.HTTPConflict(reason=f"{exc}") from exc
         except ClustersKeeperNotAvailableError as exc:
             raise web.HTTPServiceUnavailable(reason=f"{exc}") from exc
+        except ProjectNodeRequiredInputsNotSetError as exc:
+            raise web.HTTPConflict(
+                reason=f"{exc}",
+                text=f"{exc}",
```

Comment:
should only fill `reason` with a "human readable message"

The rest is taken care by the middleware. In PR https://github.com/ITISFoundation/osparc-simcore/pull/5487 this will be further simplified.

Reply:
also why do you need to add all these ```text``` and ```content_type``` stuff if the others don't? is this taken care of already in the middleware @pcrespov?